cmd/agent/container/setup.go

```diff
@@ -318,14 +319,19 @@ func dockerlessBuild(
 	}
 
 	// write output to log
-	writer := log.Writer(logrus.InfoLevel, false)
-	defer writer.Close()
+	errWriter := log.Writer(logrus.InfoLevel, false)
```

Comment:
should this be `logrus.ErrLevel`? Logging erros as info seems confusing. If we can't log as err, can we prepend a log message saying something like `error from ...`?

Reply:
If I'm not mistaken we get the build output through stderr